server.js

```diff
@@ -76,6 +76,7 @@ module.exports = (security, appInsights) => {
   app.use('/logout', security.logout());
   app.use('/oauth2/callback', security.OAuth2CallbackEndpoint());
   app.use('/health/liveness', (req, res) => res.status(HttpStatus.OK).json({ status: 'UP' }));
+  app.use('/health/readiness', (req, res) => res.status(HttpStatus.OK).json({ status: 'UP' }));
```

Comment:
You should ideally use https://github.com/hmcts/nodejs-healthcheck module to make these changes easy. Also read https://github.com/hmcts/nodejs-healthcheck#what-to-include-in-readiness-checks for more information

Reply:
I will address this in next PR